lib/linguist/languages.yml

```diff
@@ -3497,6 +3497,14 @@ Julia:
   codemirror_mode: julia
   codemirror_mime_type: text/x-julia
   language_id: 184
+Julia REPL:
+  aliases:
+  - julia-repl
```

Comment:
The alias is unnecessary as this what Linguist [already does](https://github.com/github-linguist/linguist/blob/e2012cd67867e6287ae5ee96c8fe0b92f9a88efa/lib/linguist/language.rb#L443), hence the test failure.

Reply:
Removed in [4fd4cec](https://github.com/github-linguist/linguist/pull/6859/commits/4fd4cec027915a31d6721530a69e301c621febeb)